specification/containerservice/Fleet.Management/fleet.tsp

```diff
@@ -50,7 +50,12 @@ model FleetProperties {
 
   #suppress "@azure-tools/typespec-providerhub/non-breaking-versioning" "https://github.com/Azure/typespec-azure/issues/3558"
   @doc("The FleetHubProfile configures the Fleet's hub.")
+  @added(Versions.v2022_09_02_preview)
+  @added(Versions.v2023_03_15_preview)
+  @added(Versions.v2023_06_15_preview)
+  @added(Versions.v2023_08_15_preview)
   @removed(Versions.v2023_10_15)
+  @added(Versions.v2024_02_02_preview)
```

Comment:
FleetHubProfile should be in 2024-04-01

Reply:
Below, both enableVnetIntegration and subnetId in APIServerAccessProfile should *not* be in 2024-04-01